app/src/forms/email/assets/status-assignment-email.html

```diff
@@ -1,111 +1,259 @@
+  <div class="WordSection1">
+    <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation" style="
+            background-color: #003366;
+            width: 100%;
+          ">
+      <tbody>
+        <tr>
+          <td style="font-size: 0px; padding: 0px; word-break: break-word">
+            <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation" style="
+          background-color: #003366;
+          border-bottom: 2px solid #fcba19;
+          width: 100%;
+        ">
+              <tbody>
+                <tr>
+                  <td>
+                    <div style="margin: 0px auto; max-width: 600px">
+                      <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation"
+                        style="width: 100%">
+                        <tbody>
+                          <tr>
+                            <td style="
+                            direction: ltr;
+                            font-size: 0px;
+                            padding: 20px 0;
+                            text-align: center;
+                            vertical-align: top;
+                          ">
+                              <div class="dys-column-per-100 outlook-group-fix" style="
+                              direction: ltr;
+                              display: inline-block;
+                              font-size: 13px;
+                              text-align: left;
+                              vertical-align: top;
+                              width: 100%;
+                            ">
+                                <table border="0" cellpadding="0" cellspacing="0" role="presentation"
+                                  style="vertical-align: top" width="100%">
+                                  <tbody>
+                                    <tr>
+                                      <td align="center" style="
+                                    font-size: 0px;
+                                    padding: 5px 25px;
+                                    word-break: break-word;
+                                  ">
+                                        <a style="text-decoration: none;" href="https://subscribe.developer.gov.bc.ca">
+                                          <div style="
+                                          color: #ffffff;
+                                          font-family: Oxygen, Helvetica neue, sans-serif;
+                                          font-size: 24px;
+                                          font-weight: 700;
+                                          line-height: 37px;
+                                          text-decoration: none;
+                                          display: block;
+                                        ">
+                                            {{ title }}
+                                          </div>
+                                        </a>
```

Comment:
This anchor attribute should not href to subscribe.developer.gov.bc.ca. Since we don't have a clear concept between user and staff for CHEFS yet, I'm going to suggest we remove the hyperlink from this entirely; instead just convert this to a div containing the title and it should be sufficient.